addons/repair/models/stock_picking.py

```diff
@@ -74,7 +85,7 @@ def _compute_count_repair(self):
     @api.depends('return_type_of_ids', 'code')
     def _compute_is_repairable(self):
```

Comment:
Please correct me if wrong! But wasn't it the case that `compute` methods should *always* set a value? Shouldn't this method have a "fallback" value if the condition is not met?

Reply:
The idea here is to force the value to False (default) whenever conditions for configuration of this field are no more met.
In other case we just want to keep the user-set value, I find it kind of useless to reassign it's own value to the variable ^_^'

```
else:
    picking_type.is_repairable = picking_type.is_repairable
```